backend/coreapp/views.py

```diff
@@ -337,3 +343,29 @@ def user(request, username):
     """
 
     return Response(serialize_profile(request, get_object_or_404(Profile, user__username=username)))
+
+def export(request, slug):
+    # Double-check `slug` to prevent some injection attacks
+    validate_slug(slug)
+    scratch = get_object_or_404(Scratch, slug=slug)
+
+    metadata = ScratchWithMetadataSerializer(scratch, context={ "request": request }).data
+    metadata.pop("source_code")
+    metadata.pop("context")
```

Comment:
Not sure if I should create a new `Serializer` for this specific case?

Reply:
I think this is fine